bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -71,6 +71,7 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
             dataset_id='{{ task.dataset }}',
             project_id='{{ task.project }}',
             {%+ endif -%}
+            is_dq_check_fail = {{ task.is_dq_check_fail }},
```

Comment:
This line is not required and can be removed since the `bigquery_dq_check` function call has not changed.

Reply:
I guess Airflow needs to know whether to run the `warn` or `fail` checks. I'll add it to the function in telemetry-airflow

Reply:
see https://github.com/mozilla/telemetry-airflow/pull/1793